src/currencyEngineETH.js

```diff
@@ -1307,7 +1307,9 @@ class EthereumEngine {
     }
 
     const InsufficientFundsError = new Error('Insufficient funds')
-    InsufficientFundsError.name = 'InsufficientFundsError'
+    InsufficientFundsError.name = 'ErrorInsufficientFunds'
+    const InsufficientFundsEthError = new Error('Insufficient funds. Need more ETH')
```

Comment:
Maybe change the error to something like "Insufficient ETH to fund the transaction fees" so that the user would understand why he even needs ETH when trying to transfer a token.